magpie/api/management/user/user_utils.py

```diff
@@ -112,40 +111,16 @@ def _add_to_group(usr, grp):
         _add_to_group(new_user, _get_group(anonym_grp_name))
         new_user_groups.append(anonym_grp_name)
 
-    # Check for webhook requests
-    config_path = get_constant("MAGPIE_CONFIG_PATH", default_value=None,
-                               raise_missing=False, raise_not_set=False, print_missing=True)
-    if config_path:
-        webhook_configs = get_all_configs(config_path, "webhooks", allow_missing=True)
-        for cfg in webhook_configs:
-            if "create" in cfg.keys() and len(cfg["create"]) > 0:
-                # Execute all webhook requests
-                pool = multiprocessing.Pool(processes=len(cfg["create"]))
-                args = [(url, user_name) for url in cfg["create"]]
-                pool.starmap_async(webhook_request, args, error_callback=webhook_error_callback)
+    user_content = uf.format_user(new_user, new_user_groups)
 
-    return ax.valid_http(http_success=HTTPCreated, detail=s.Users_POST_CreatedResponseSchema.description,
-                         content={"user": uf.format_user(new_user, new_user_groups)})
-
-
-def webhook_request(webhook_url, user_name):
-    # type: (Str, Str) -> None
-    """
-    Sends a webhook request using the input url.
-    """
-    # TODO: create a real temp_url that will be called if the webhook service has an error
-    #  this would also set the user's status to 0
-    requests.post(webhook_url, data={"user_name": user_name, "temp_url": "temp_url:80/todo"})
+    # Force commit before sending the webhook requests, so that the user's status is editable if a webhook error occurs
+    transaction.commit()
 
+    # Process any webhook requests
+    process_webhook_requests(WEBHOOK_CREATE_USER_ACTION, {"user_name": user_name, "tmp_url": "tmp_url:80/todo"}, True)
```

Comment:
tmp url should be available right now. We plan on doing another PR to complete it?

Reply:
That would be the plan, yes, since this PR started before the tmp_url was finished.